aptos-move/aptos-transactional-test-harness/tests/aptos_test_harness/diamond_clicker.v2_exp

```diff
@@ -33,26 +33,26 @@ B0:
 	9: MutBorrowField[0](OuterStruct.any_field: vector<InnerStruct>)
 	10: StLoc[2](loc1: &mut vector<InnerStruct>)
 	11: LdU64(0)
-	12: StLoc[3](loc2: u64)
```

Comment:
These changes are unrelated to this PR, but were obtained by running `move_pr.sh -i2`.